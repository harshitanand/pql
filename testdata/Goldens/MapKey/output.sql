WITH "__subquery0" AS (SELECT * FROM "MapTable" WHERE ("a"['key2']) > 10 ORDER BY "id" ASC NULLS FIRST)
SELECT "a"['key1'] AS "Key1", "a"['key2'] AS "Key2" FROM "__subquery0";
