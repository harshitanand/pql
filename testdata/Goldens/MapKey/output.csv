Key1,Key2
2,20
3,30
